config_cmake.h.in

```diff
@@ -145,6 +145,9 @@
 /* Define to the full name of this package. */
 #define PACKAGE_NAME "fish"
 
+/* Define to 1 if tparm accepts a fixed amount of parameters. */
+#cmakedefine TPARM_VARARGS 1
```

Comment:
1. The comment has it the wrong way around

2. This doesn't appear to be used anywhere, or does NetBSD curses use it?

Reply:
It changes the behaviour of the NetBSD curses headers if defined.

Reply:
Fixed, thanks!